### DISCUSSION TITLE
What a course!!

### DISCUSSION THREAD

**Discussion by adtygan** [OpenMined/courses#196] [Feedback] [answered]
Just finished the course yesterday and I got to say this - this course opened a lot of new ideas for me. But I would like to thank the team for one specific thing - for reiterating lessons all along the way. It might come off as a trivial thing to repeat the stuff being taught at different points in the course. But for me, it has helped retain the crux of the course and I'm happy the team behind the course made it happen for me. Thanks a lot again. I'm eagerly waiting for the next course. 

Just one suggestion: I would be happy if you could put the important points in text on the screen for the upcoming course. Especially for things like formulae, it would be great to put it on the screen as a text to reinforce the idea even more.

**Comment by yemikifouly** [comment, marked as answer]
Aditya, I am so glad that you enjoyed the course. And thank you very much for the glowing feedback! Feedback like this one make our day! 🙂🙌 About your suggestion, great point!!! This is already in the works actually. In the coming weeks, text will be added to the videos highlighting concepts presented in the videos. 🙂 

Again, thank you so much for the feedback. How are you doing with the project? Got an idea yet?

**Reply by adtygan** [reply]
I submitted my project yesterday. I read in another discussion that mentors are currently being trained, so I'm waiting for the response. Was actually fun writing the proposal xD.

**Reply by yemikifouly** [reply]
Indeed... Project reviews should start towards the end of this week. 🙂 Having fun doing a project is a rare thing. It is great that you had fun writing the proposal! 😃
